packages/cmf/src/components/Saga/CmfRegisteredSaga.component.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { v4 } from 'uuid';
+import cmfConnect from '../../cmfConnect';
+import { start, stop } from '../../actions/saga';
+
+export function CmfRegisteredSagaComponent({
+	dispatch,
+	sagaId,
+	sagaAttributes,
+	componentId = 'default',
+	children = null,
+}) {
+	// useState is used to keep the v4 value
+	const [id] = useState(v4());
```

Comment:
I would suggest to use `useRef` instead of `useMemo`. 

From [official doc](https://reactjs.org/docs/hooks-reference.html#usememo): 
`You may rely on useMemo as a performance optimization, not as a semantic guarantee. In the future, React may choose to “forget” some previously memoized values and recalculate them on next render, e.g. to free memory for offscreen components.`

So I think it is safer to go with a `useRef`. Wdyt?